packages/dw-react-utils/src/stories/hooks/useDrag.stories.tsx

```diff
@@ -62,6 +63,35 @@ export const Horizontal: Story = () => {
   )
 }
 
+export const Disable: Story = () => {
+  const dragRef = React.useRef() as React.MutableRefObject<HTMLDivElement>;
+  const [dragElement, setDragElement] = React.useState<HTMLElement>();
+	const [disabled, setIsDisabled] = React.useState(false);
+  const { isDragging, positionX, positionY } = useDrag(dragElement, "horizontal", undefined, disabled);
+
+	const onChange = (e: React.FormEvent<HTMLInputElement>) => {
```

Comment:
another weird spacing (i'm beginning to suspect something wrong in GitHub rendering :p)